pkg/api/admin/openshiftcluster.go

```diff
@@ -34,6 +34,7 @@ type OpenShiftClusterProperties struct {
 	LastProvisioningState           ProvisioningState       `json:"lastProvisioningState,omitempty"`
 	FailedProvisioningState         ProvisioningState       `json:"failedProvisioningState,omitempty"`
 	LastAdminUpdateError            string                  `json:"lastAdminUpdateError,omitempty"`
+	MaintenanceTask                 MaintenanceTask         `json:"maintenanceTask,omitempty"`
```

Comment:
As in, once we've set it?

Reply:
This might prevent us from changing this field via admin API:

https://github.com/Azure/ARO-RP/blob/fef8f41aa37faa8bae0e27ea896a40a549cd9d3a/pkg/api/admin/openshiftcluster_validatestatic.go#L26-L30

I can imagine a case of update failing with `maintenanceTask` set to `OperatorUpdate`. How do we fix it? Will we have to manually edit a doc in the DB and remove this field?

Reply:
OHHHHHHH

I get you now. I thought you were asking if it had to be mutable, in "should make it immutable", I forgot that it was immutable by default.

Fixed.